### PR TITLE
Add single quote in change-perfdata help

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

**PLEASE MAKE SURE THAT THE BRANCH PR INCLUDES JIRA TICKET ID** (_for centreon-internal_)

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by omercier — DISMISSED** [review]

**Review by lucie-dubrunfaut — APPROVED** [review]

File: .github/actions/promote-to-stable/action.yml [hunk 1]
```diff
@@ -41,16 +41,20 @@ runs:
...
-            echo "[DEBUG] - Promoting $ARTIFACT to stable."
-            jf rt cp $ARTIFACT $TARGET_PATH --flat=true
+            echo "[DEBUG] - Downloading $ARTIFACT from TESTING."
+            jf rt download $ARTIFACT --flat
+          done
+          for ARTIFACT_DL in $(dir|grep -E "*.rpm"); do
+            echo "[DEBUG] - Promoting (upload) $ARTIFACT_DL to stable $TARGET_PATH."
+            jf rt upload "$ARTIFACT_DL" "$TARGET_PATH" --flat
           done
+          rm -f *.rpm
```
Comment (sc979) [inline .github/actions/promote-to-stable/action.yml:57]: Hi @YanMeddour,
These modifications are not part of your PR 🤔

File: .github/workflows/plugins.yml [hunk 2]
```diff
@@ -141,7 +141,7 @@ jobs:
     steps:
       - name: Checkout sources
         if: ${{ matrix.distrib == 'el7' }}
-        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        uses: actions/checkout@v3 # el7 is not compatible with checkout v4 which uses node20
```
Comment (sc979) [inline .github/workflows/plugins.yml:144]: This has already been merged on develop

**Review by sc979 — CHANGES_REQUESTED** [review]
Hi @YanMeddour ,

Kindly, could you rebase your branch on develop ?
`git rebase origin/develop`
Some commits have already been merged and should not be displayed 🤔

File: .github/actions/promote-to-stable/action.yml (hunk 1 above)
Comment (lucie-dubrunfaut) [reply .github/actions/promote-to-stable/action.yml:57]: Yes totally my bad I tried to rebase this PR in aim to help the validation of it but it looks it goes wrong :<
@omercier and me will check about this next week

**Review by lucie-dubrunfaut — DISMISSED** [review]

File: src/centreon/plugins/output.pm [hunk 3]
```diff
@@ -1539,29 +1539,29 @@ remove all metrics whose value equals 0 and that don't have a maximum value.
...
+Convert storage free perfdata into used: --change-perfdata='free,used,invert()'
 
-Convert storage free perfdata into used: --change-perfdata=used,free,invert()
+Convert storage free perfdata into used: --change-perfdata='used,free,invert()'
 
-Scale traffic values automatically: --change-perfdata=traffic,,scale(auto)
+Scale traffic values automatically: --change-perfdata='traffic,,scale(auto)'
 
-Scale traffic values in Mbps: --change-perfdata=traffic_in,,scale(Mbps),mbps
+Scale traffic values in Mbps: --change-perfdata='traffic_in,,scale"Mbps",mbps'
```
Comment (omercier) [inline src/centreon/plugins/output.pm:1562]:
```suggestion
Scale traffic values in Mbps: --change-perfdata='traffic_in,,scale(Mbps),mbps'
```

**Review by sdepassio — APPROVED** [review]

**Review by sc979 — APPROVED** [review]

**Pull request merged**